<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>historyWindow</class>
 <widget class="QMainWindow" name="historyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1351</width>
    <height>1036</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="chart_view_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1151</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="chart_view_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>1151</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_comp_button">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>960</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="settings_comp_button">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>930</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="Line" name="midline">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>1311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="year_comp_button_1">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>240</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Year</string>
    </property>
   </widget>
   <widget class="QLabel" name="gas_comp_label_1">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>110</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Gas:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="co2_comp_check_1">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>130</y>
      <width>72</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>CO2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="so2_comp_check_1">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>150</y>
      <width>72</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>SO2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="week_comp_button_1">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>300</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Week</string>
    </property>
   </widget>
   <widget class="QComboBox" name="station_comp_combo_1">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>40</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>MeasuringStation1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MeasuringStation2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MeasuringStation3</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="max_comp_check_1">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>140</y>
      <width>72</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>max.</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="avg_comp_check_1">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>180</y>
      <width>72</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>avg.</string>
    </property>
   </widget>
   <widget class="Line" name="v_line_1">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>90</y>
      <width>16</width>
      <height>131</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="nox_comp_check_1">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>170</y>
      <width>72</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>NOx</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="other_comp_check_1">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>190</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>(other)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="data_comp_combo_1">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>10</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>SMEAR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>STATFI</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="date_to_comp_button_1">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>360</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Date-to-Date</string>
    </property>
   </widget>
   <widget class="QPushButton" name="day_comp_button_1">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>330</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>24h</string>
    </property>
   </widget>
   <widget class="QPushButton" name="month_comp_button_1">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>270</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Month</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="min_comp_check_1">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>160</y>
      <width>72</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>min.</string>
    </property>
   </widget>
   <widget class="QLabel" name="data_comp_label_1">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>110</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Additional data:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="year_comp_button_2">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>680</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Year</string>
    </property>
   </widget>
   <widget class="QLabel" name="gas_comp_label_2">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>550</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Gas:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="co2_comp_check_2">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>570</y>
      <width>72</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>CO2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="so2_comp_check_2">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>590</y>
      <width>72</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>SO2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="week_comp_button_2">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>740</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Week</string>
    </property>
   </widget>
   <widget class="QComboBox" name="station_comp_combo_2">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>480</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>MeasuringStation1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MeasuringStation2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MeasuringStation3</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="max_comp_check_2">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>580</y>
      <width>72</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>max.</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="avg_comp_check_2">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>620</y>
      <width>72</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>avg.</string>
    </property>
   </widget>
   <widget class="Line" name="v_line_2">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>530</y>
      <width>16</width>
      <height>131</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="nox_comp_check_2">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>610</y>
      <width>72</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>NOx</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="other_comp_check_2">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>630</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>(other)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="data_comp_combo_2">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>450</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>SMEAR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>STATFI</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="date_to_comp_button_2">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>800</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Date-to-Date</string>
    </property>
   </widget>
   <widget class="QPushButton" name="day_comp_button_2">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>770</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>24h</string>
    </property>
   </widget>
   <widget class="QPushButton" name="month_comp_button_2">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>710</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Month</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="min_comp_check_2">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>600</y>
      <width>72</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>min.</string>
    </property>
   </widget>
   <widget class="QLabel" name="data_comp_label_2">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>550</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Additional data:</string>
    </property>
   </widget>
   <widget class="Line" name="h_line_2">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>870</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="default_comp_button_2">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>830</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Default loadout</string>
    </property>
   </widget>
   <widget class="QPushButton" name="default_comp_button_1">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>390</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Default loadout</string>
    </property>
   </widget>
   <widget class="QPushButton" name="default_all_button">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>900</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Default all</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1351</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWeather_Report">
    <property name="title">
     <string>Weather Report</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="action_pdf"/>
     <addaction name="action_PNG"/>
     <addaction name="action_JPEG"/>
    </widget>
    <addaction name="menuExport"/>
    <addaction name="actionSave_as"/>
   </widget>
   <addaction name="menuWeather_Report"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="action_pdf">
   <property name="text">
    <string>.pdf</string>
   </property>
  </action>
  <action name="action_PNG">
   <property name="text">
    <string>.PNG</string>
   </property>
  </action>
  <action name="action_JPEG">
   <property name="text">
    <string>.JPEG</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
